### PR TITLE
Various minor updates

### PR DESCRIPTION
**Pull request by titusfortner**
### Description
* Update installation & contributing docs
* Change the text and bold the link to the full documentation
* Add implementation badge to indicate features that aren't available in that language yet
* Add examples badge to indicate that hard coded examples need to be moved over to github (addresses concern in - https://github.com/SeleniumHQ/seleniumhq.github.io/pull/1358?notification_referrer_id=NT_kwDOAAvb4bE2MTQ0Njk5NjExOjc3NzE4NQ#issuecomment-1593179225)
* If using --ignoreCache it can take longer than default 30 seconds to start up

### Motivation and Context

Rather than a bunch of little PRs or merging directly, here's one PR for feedback on a number of things.

The big change here is how we display the link to the code on GitHub

### CODE REVIEW

**Comment by titusfortner** [comment]
Here's a good example of the bold note about seeing full code - https://deploy-preview-1406--selenium-dev.netlify.app/documentation/webdriver/drivers/service/#default-service-instance

File: README.md [hunk 1]
```diff
@@ -16,6 +16,7 @@ to use Hugo 0.110.0.
 Steps needed to have this working locally and work on it:
 
 - Follow the [Install Hugo](https://www.docsy.dev/docs/get-started/other-options/#install-hugo) instructions from Docsy
+- [Install go](https://go.dev/doc/install)
```
Comment (harsha509) [inline README.md:19]: hi 
just curious, is the `Go` being used for any specific purpose within our docs?

Comment (titusfortner) [reply README.md:19]: It's required now that we moved to the module system with docsy.

Comment (diemol) [reply README.md:19]: https://gohugo.io/installation/macos/#prerequisites

**Review by diemol — APPROVED** [review]
Thank you, @titusfortner!

**Pull request merged**